### PR TITLE
Make `scripts/*.cr` all executable

### PR DESCRIPTION
**Pull request by HertzDevil**
Adds the execute permission bits and `#!`.

### CODE REVIEW

File: scripts/generate_grapheme_break_specs.cr [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#! /usr/bin/env crystal
```
Comment (Sija) [inline scripts/generate_grapheme_break_specs.cr:1]: There's no need for space, ditto elsewhere

```suggestion
#!/usr/bin/env crystal
```

Comment (HertzDevil) [reply scripts/generate_grapheme_break_specs.cr:1]: There are already spaces in the remaining files under `scripts/`

**Comment by straight-shoota** [comment]
Some CI jobs are failing

**Comment by HertzDevil** [comment]
`make crystal` shouldn't regenerate the data, both here and in #13998. Do you have an idea why this happens?

**Comment by HertzDevil** [comment]
I am now quite sure that neither this nor #13998 caused the CI failures because it is now happening to entirely unrelated PRs: https://github.com/crystal-lang/crystal/actions/workflows/aarch64.yml

**Comment by straight-shoota** [comment]
Let's discuss that CI bug in #14007

I suppose we can proceed with this then...

**Review by straight-shoota — APPROVED** [review]

**Pull request merged**